config/opa-service/rego/constraints.rego

```diff
@@ -0,0 +1,31 @@
+package envoy.authz
```

Comment:
cool, but what about e.g. boolean operators? e.g. what would I do if I wanted to express a constraint like `(x < 5 && x > 10) || x > 20`...